features/apimgt/org.wso2.carbon.apimgt.store.feature/src/main/resources/store-new/source/src/app/components/Shared/AppsAndKeys/WaitingForApproval.jsx

```diff
@@ -0,0 +1,21 @@
+import React from 'react';
+import { withStyles } from '@material-ui/core/styles';
+
+const styles = theme => ({
+    root: {
+        padding: theme.spacing.unit * 3,
+    },
+});
+
+const waitingForApproval = (props) => {
+    const { classes, keyState, states } = props;
+    let message = 'A request to register this application has been sent.';
```

Comment:
shouldn't this message be internationalized?

Reply:
@dushansilva lets note to fix during hackathon

Reply:
noted